src/styles/hover-menu.scss

```diff
@@ -70,6 +70,7 @@
           font-family: inherit;
           font-feature-settings: inherit;
           font-weight: 500;
+          width: 100%;
```

Comment:
This is unrelated, but fixes the issue where clicking on the 'unlink account' button from the three-dot dropdown seems to not work because you're not clicking the text.